api/c/bingo-nosql/src/bingo_matcher.h

```diff
@@ -517,8 +517,8 @@ namespace bingo
         }
 
     protected:
-        void _setParameters(const char* params) override {};
-        void _initPartition() override {};
+        void _setParameters(const char* params) override{};
+        void _initPartition() override{};
```

Comment:
IMHO, this is wrong format change.

Reply:
It was the same before, but fixed.

Reply:
Fixed by moving the body to .cpp file...
I meant the space before the opening brace in such constructions where the empty body is shortened to `{}`
Is there a rule for this cases?